app/notifications/notifications_letter_callback.py

```diff
@@ -177,8 +171,8 @@ def extract_properties_from_request(request_data) -> LetterUpdate:
     postage = next(item["value"] for item in despatch_properties if item["key"] == "postageClass")
     cost_threshold = _get_cost_threshold(mailing_product, postage)
 
-    despatch_datetime = next(item["value"] for item in despatch_properties if item["key"] == "productionRunDate")
-    despatch_date = _get_despatch_date(despatch_datetime)
+    despatch_datetime = request_data["data"]["transitionDate"]
+    despatch_date = datetime.datetime.strptime(despatch_datetime, "%Y-%m-%dT%H:%M:%SZ").date()
```

Comment:
if it's an iso timestamp we can use `datetime.fromisoformat`

however, we then need to make sure we get a date from it - we'll probs need to use `convert_utc_to_bst` from notifications_utils.timezone

(and make sure you add a test first to prove that it's currently buggy if we get a callback at 23:30 utc during summertime